src/templates/static/affiliates/faq.jsx

```diff
@@ -80,7 +80,7 @@ const FAQ = () => (
 
                 <Section id='marketing-and-promotions' header={it.L('Marketing and promotions')}>
                     <h3>{it.L('What advertising materials and marketing tools do you offer?')}</h3>
-                    <p>{it.L('We have a tested and proven selection of advertising materials including banners, reviews, videos, and text ads for you to use. If you would like certain items to be customised to your preferences, please contact your account manager at <a href=\'[_1]\'>affiliates@binary.com</a>. You can also use the Myaffiliates system to track and optimise your campaigns.', 'mailto:affiliates@binary.com')}</p>
+                    <p>{it.L('We have a tested and proven selection of advertising materials including banners, reviews, videos, and text ads for you to use. If you would like certain items to be customised to your preferences, please contact your account manager at <a href=\'[_1]\'>[_2]</a>. You can also use the Myaffiliates system to track and optimise your campaigns.', `mailto:${it.affiliate_email}`, it.affiliate_email)}</p>
```

Comment:
then why not change the whole `<a href=\'[_1]\'>[_2]</a>` to `[_1]` and replace it with something like `<a href="mailto:${it.affiliate_email}">it.affiliate_email</a>`. then translators don't have to deal with tags either. both here and line 104